nexus-repository-helm/src/main/java/org/sonatype/repository/helm/internal/content/recipe/HelmHostedFacetImpl.java

```diff
@@ -0,0 +1,69 @@
+package org.sonatype.repository.helm.internal.content.recipe;
+
+import java.io.IOException;
+
+import javax.annotation.Nullable;
+import javax.inject.Inject;
+import javax.inject.Named;
+
+import org.sonatype.nexus.repository.FacetSupport;
+import org.sonatype.nexus.repository.config.Configuration;
+import org.sonatype.nexus.repository.view.Content;
+import org.sonatype.nexus.repository.view.Payload;
+import org.sonatype.repository.helm.internal.AssetKind;
+import org.sonatype.repository.helm.internal.content.HelmContentFacet;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+import static org.sonatype.repository.helm.internal.AssetKind.HELM_PACKAGE;
+import static org.sonatype.repository.helm.internal.AssetKind.HELM_PROVENANCE;
+
+/**
+ * {@link HelmHostedFacet implementation}
+ *
+ * @since 1.0.11
+ */
+@Named
+public class HelmHostedFacetImpl
+    extends FacetSupport
+    implements HelmHostedFacet
+{
+  private HelmContentFacet helmContentFacet;
+
+  @Inject
+  public HelmHostedFacetImpl()
+  {
```

Comment:
You can add no-op comment here.

Reply:
Fixed 134e0fecc36375934a081e33e7ea5d90e2c92183